test/unstable-keyword/config/const/unstable_config_const.compopts

```diff
@@ -0,0 +1,3 @@
+ # unstable_config_const.unset.good
+--no-warn-unstable # unstable_config_const.unset.good
+--warn-unstable # unstable_config_const.set.good
```

Comment:
So, configs are a little trickier than other symbols.  In addition to generating warnings when they are accessed in the code, they also should generate warnings when they are set via compilation and execution flags.  Note that the tests in the deprecated-keyword directory had .execopts or .compopts files for the config cases which set these symbols using the flags.

I think it'll be a little tricky to interweave those settings with `--warn-unstable`.  You'll end up with the cases ballooning and `start_test` doesn't like specifying a .good in both the .compopts and .execopts for the same test (it doesn't know which one should win and honestly, it would be tricky to determine it).  I'd probably do something like split into separate tests:
- one that uses the default setting for `--warn-unstable` and varies based on whether the config is set
- one that always uses `--no-warn-unstable` and varies based on whether the config is set
- and one that always uses `--warn-unstable` and varies based on whether the config is set

Feel free to put this change off until the very end and let me handle it if you run out of time, since it'll be finicky.